packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerDelegate.java

```diff
@@ -447,14 +446,18 @@ public void onPathReady(String path) {
   }
 
   private void handleImageResult(String path) {
-    if (pendingResult != null) {
-      Double maxWidth = methodCall.argument("maxWidth");
-      Double maxHeight = methodCall.argument("maxHeight");
-
-      String finalImagePath = imageResizer.resizeImageIfNeeded(path, maxWidth, maxHeight);
-      finishWithSuccess(finalImagePath);
-    } else {
-      throw new IllegalStateException("Received image from picker that was not requested");
+    try{
```

Comment:
This code doesn't look relevant to fingerprint functionality. Is this change here intentionally?